Missing encoder release compatible with parser 0.7

From @yanns in https://discord.com/channels/1022972389463687228/1117957327216201758/1163487327226712235, via https://github.com/apollographql/apollo-rs/issues/469#issuecomment-1764646902:

> Another issue upgrading `apollo-parser` to 0.7.1 is the missing update for `apollo-encoder`:
> ```rust
> error[E0277]: the trait bound `apollo_encoder::SchemaDefinition: From<apollo_parser::cst::SchemaDefinition>` is not satisfied
>   --> src/feature_toggle/schema_transformation.rs:42:68
>    |
> 42 |             Definition::SchemaDefinition(e) => pub_schema.schema(e.try_into()?),
>    |                                                                    ^^^^^^^^ the trait `From<apollo_parser::cst::SchemaDefinition>` is not implemented for `apollo_encoder::SchemaDefinition`
>    |
>    = note: required for `apollo_parser::cst::SchemaDefinition` to implement `Into<apollo_encoder::SchemaDefinition>`
>    = note: required for `apollo_encoder::SchemaDefinition` to implement `TryFrom<apollo_parser::cst::SchemaDefinition>`
>    = note: required for `apollo_parser::cst::SchemaDefinition` to implement `TryInto<apollo_encoder::SchemaDefinition>`
> ```

--- a/crates/apollo-encoder/CHANGELOG.md
+++ b/crates/apollo-encoder/CHANGELOG.md
@@ -17,21 +17,19 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## Maintenance
 
 ## Documentation -->
-# [0.8.0] (unreleased) - 2023-mm-dd
+# [0.8.0](https://crates.io/crates/apollo-encoder/0.8.0) - 2023-10-19
 
 ## BREAKING
-- **apollo-parser@0.7.0 - [SimonSapin], [pull/???]**
+- **apollo-parser@0.7.0 - [SimonSapin]**
 
   This updates the version of `apollo-parser` required by the `TryFrom`
   implementations in this crate.
 
-- **apollo-compiler@0.12.0 - [SimonSapin], [pull/???]**
-
-  This updates the version of `apollo-compiler` required by the `TryFrom`
-  implementations in this crate.
+- **removes `TryFrom` from apollo-compiler - [SimonSapin]**
+  `apollo-compiler@1.0.0` can be directly serialised to SDL without requiring
+  apollo-encoder. The `TryFrom` implementation is therefore no longer necessary.
 
 [SimonSapin]: https://github.com/SimonSapin
-[pull/???]: https://github.com/apollographql/apollo-rs/pull/???
 
 # [0.7.0](https://crates.io/crates/apollo-encoder/0.7.0) - 2023-08-18
 
@@ -73,11 +71,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## Features
 - **add `Clone` impl to encoder types - [zackangelo], [pull/515]**
-  Consistently derive `Clone` on all encoder types.  
+  Consistently derive `Clone` on all encoder types.
 
   [zackangelo]: https://github.com/zackangelo
   [pull/515]: https://github.com/apollographql/apollo-rs/pull/515
-  
+
 # [0.5.0](https://crates.io/crates/apollo-encoder/0.5.0) - 2023-02-17
 
 ## BREAKING

--- a/crates/apollo-encoder/Cargo.toml
+++ b/crates/apollo-encoder/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "apollo-encoder"
-version = "0.7.0" # When bumping, also update README.md
+version = "0.8.0" # When bumping, also update README.md
 authors = [
     "Irina Shestak <shestak.irina@gmail.com>",
     "Benjamin Coenen <benjamin.coenen@apollographql.com>",

--- a/crates/apollo-encoder/README.md
+++ b/crates/apollo-encoder/README.md
@@ -28,7 +28,7 @@ Or add this to your `Cargo.toml` for a manual installation:
 ```toml
 # Just an example, change to the necessary package version.
 [dependencies]
-apollo-encoder = "0.7.0"
+apollo-encoder = "0.8.0"
 ```
 
 ## Rust versions

--- a/crates/apollo-parser/Cargo.toml
+++ b/crates/apollo-parser/Cargo.toml
@@ -24,7 +24,7 @@ thiserror = "1.0.30"
 [dev-dependencies]
 indexmap = "2.0.0"
 miette = { version = "3.2.0", features = ["fancy"] }
-apollo-encoder = { path = "../apollo-encoder", version = "0.7.0", features = [
+apollo-encoder = { path = "../apollo-encoder", version = "0.8.0", features = [
     "apollo-parser",
 ] }
 anyhow = "1.0.66"

--- a/crates/apollo-smith/Cargo.toml
+++ b/crates/apollo-smith/Cargo.toml
@@ -24,7 +24,7 @@ categories = [
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-apollo-encoder = { path = "../apollo-encoder", version = "0.7.0" }
+apollo-encoder = { path = "../apollo-encoder", version = "0.8.0" }
 apollo-parser = { path = "../apollo-parser", version = "0.7.0", optional = true }
 arbitrary = { version = "1.3.0", features = ["derive"] }
 once_cell = "1.9.0"

